handbook/customer-success/README.md

```diff
@@ -34,6 +34,22 @@ Before a routine customer call, the CSM prepares an agenda including the followi
 5. Provide updates to open feature requests (can be done monthly or quarterly)
 6. Provide updates to open bug reports
 
+### Generate an expansion opportunity in Salesforce
+Customer Success Manager's (CSM's) are responsible for developing customer expansion opportunities that are not being worked on in conjunction with an Account Executive (AE). An AE will be assigned by the Chief Revenue Officer (CRO) for large scale expansion opportunities such as bringing on a new Fleet use case or bringing on a new group of hosts to an existing Fleet use case. CSM's manage expansion opportunities for things like host count increase for customer growth and price increases on renewals. Discuss examples of these scenarios with your manager to learn more.
+
+Take the steps below when creating an expansion opportunity in Salesforce:
+
+1. Navigate to the customer account record in Salesforce.
+2. Scroll down to the section labeled "Upcoming renewal (± expansions)".
+3. Click the "New" button.
+4. Change the type of opportunity to "Expansion".
+5. Change the opportunity name to reflect the following naming structure: CustomerName_FleetProduct_Expansion_QuarterDue
+    - Example: ABCTestCompany_FleetPremiumMDM_HostExpansion_Q12025
+    - Example: ABCTestCompany_FleetPremium_PriceIncreaseExpansion_Q12025
+6. Fill out all the required fields.
+7. Press the "Save" button to complete the opportunity creation process.
+8. Moving forward, CSM's are responsible for keeping the stage, next steps, and date of next steps fields updated as the opportunity progresses through the sales cycle.
```

Comment:
This isn't actually a step that needs to be done in order. Let's move it up top in the preamble.